server-embedded/src/main/java/org/exoplatform/chat/services/ChatServiceImpl.java

```diff
@@ -166,6 +173,28 @@ public void write(String clientId, String message, String sender, String room, S
       }
       notificationService.setNotificationsAsRead(sender, "chat", "room", room);
     }
+    messageInfo.put("room",room);
```

Comment:
this will not work in [standalone chat server mode](https://docs.exoplatform.org/en/6.1/Clustering.html?highlight=standalone#how-to-install-exo-chat-in-standalone-mode)
I think it's better to send this metric using [UI Watcher]( https://github.com/exoplatform/analytics/blob/develop/README.md) (See DOM Watcher examples that uses websocket https://github.com/exoplatform/analytics/blob/develop/analytics-listeners/src/main/java/org/exoplatform/analytics/listener/websocket/WebSocketUIStatisticListener.java )